apps/web/app/page.tsx

```diff
@@ -1,15 +1,24 @@
 // 3rd party
 import { JSX } from "react";
 // repo
-import Link from "@repo/ui/link";
 import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
+import WalkthroughCard from "@repo/ui/walkthrough-card";
+import { WalkthroughJSONData } from "../../../packages/data/src/useWalkthroughData";
 
 export default function Page(): JSX.Element {
   return (
     <div className="u-container">
```

Comment:
Should this div be a section? 
```
<section> 
   <div>
```

Reply:
The landing page html will be done with https://hous-bssb.atlassian.net/jira/software/c/projects/HOUSNAV/boards/3/backlog?selectedIssue=HOUSNAV-30